docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+version: '2'
+services:
+  app:
+    build: .
+    ports:
+      - 3000:3000
+    command: bash -c " npm start"
```

Comment:
Do you think we should define `volumes` in here too, to mount the current directory for development?